frontend/src/metabase/query_builder/components/filters/pickers/SelectPicker.jsx

```diff
@@ -117,7 +118,7 @@ export default class SelectPicker extends Component {
                   >
                     <CheckBox
                       checked={checked.has(option.key)}
-                      checkedColor="accent2"
+                      checkedColor={checkedColor ?? "filter"}
```

Comment:
How do I render this `<SelectPicker />` component? Just want to see what the color will look like.

Reply:
No matter what I tried, I can't seem to render `<SelectPicker />`
![image](https://user-images.githubusercontent.com/1937582/179006045-97269919-939d-4940-8bb8-414fc617d4ff.png)
![image](https://user-images.githubusercontent.com/1937582/179006129-7d1a3772-8dbe-4e26-9620-fe9267721a1a.png)

Note that when I searched for `SelectPicker` in React devtool there was 0 result (`0 | 0`).

Can you please describe the exact step to make it renders? I was using People from sample database by the way.

Reply:
oh you're right! I forgot this actually renders through `fieldValuesWidget` - I also couldn't figure out how to render `SelectPicker`

![Screen Shot 2022-07-14 at 8 41 14 AM](https://user-images.githubusercontent.com/30528226/179009392-36c0ce67-5456-4912-9d41-edff2654eefc.png)

I'll revert that change